src/adapters/rhythmone.js

```diff
@@ -0,0 +1,360 @@
+/*
+  Copyright (c) 2016 RhythmOne, LLC. All rights reserved.
```

Comment:
Please remove, all submissions are subject to the Apache 2.0 License.
https://github.com/prebid/Prebid.js/blob/master/LICENSE